src/main/java/com/blackducksoftware/integration/jira/common/BlackDuckDataHelper.java

```diff
@@ -108,20 +104,23 @@ public ProjectVersionWrapper getProjectVersionWrapper(final NotificationContentD
 
     public ProjectVersionWrapper getProjectVersionWrapper(final VersionBomComponentView versionBomComponent) throws IntegrationException {
         // TODO Stop using this when Black Duck supports going back to the project-version
-        final String versionBomComponentHref = blackDuckService.getHref(versionBomComponent);
-        final int componentsIndex = versionBomComponentHref.indexOf(ProjectVersionView.COMPONENTS_LINK);
-        final String projectVersionUri = versionBomComponentHref.substring(0, componentsIndex - 1);
+        final Optional<String> versionBomComponentHref = versionBomComponent.getHref();
+        if (versionBomComponentHref.isPresent()) {
+            final int componentsIndex = versionBomComponentHref.get().indexOf(ProjectVersionView.COMPONENTS_LINK);
```

Comment:
We call the get method twice here.  We should store it in a variable or preferrably use a map method which will remove the isPresent and get method calls.  Might clean up the code a bit here.  This would be perhaps a todo going forward since we are trying to get the existing code working with the new library.